io.github.rafaelfassi.QLogExplorer.yaml

```diff
@@ -0,0 +1,33 @@
+app-id: io.github.rafaelfassi.QLogExplorer
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+rename-appdata-file: qlogexplorer.appdata.xml
+rename-desktop-file: qlogexplorer.desktop
+rename-icon: qlogexplorer
+command: qlogexplorer
+finish-args:
+  # Required for Qt render.
+  - --socket=wayland
+  - --device=dri
+  # Necessary if X11 fallback takes place.
+  - --socket=fallback-x11
+  - --share=ipc
+  # Necessary to open system log files.
+  - --filesystem=host
```

Comment:
are you sure you can view them?

Reply:
Yeah, the user can open whatever file he has read permission.

Reply:
If this is about accessing system logs, I would change it to `--filesystem=host` and `--filesystem=/var/log`.